code/game/machinery/_machinery.dm

```diff
@@ -535,6 +535,23 @@ Class Procs:
 		return FALSE
 	return TRUE
 
+// Power cell in hand replacement
+/obj/machinery/attackby(obj/item/C, mob/user)
+	if(istype(C, /obj/item/stock_parts/cell) && panel_open)
+		locate(/obj/item/circuitboard/machine) in component_parts
+		for(var/obj/item/P in component_parts)
+			if(istype(P,/obj/item/stock_parts/cell))
+				if(user.transferItemToLoc(C, src))
+					user.put_in_active_hand(P)
+					component_parts+=C
+					component_parts-=P
+					RefreshParts()
+					playsound(src, 'sound/surgery/taperecorder_close.ogg', 50, FALSE)
+					to_chat(user, "<span class='notice'>you replaced [capitalize(P.name)]  with [C.name] via power cell socket.</span>")
```

Comment:
```suggestion
					to_chat(user, "<span class='notice'>You replace [P.name] with [C.name].</span>")
```
Should be in present tense, capitalisation is inconsistent and in the middle of the sentance, capitalisation is missed at the start of the sentence.